magento/magento2
# Increase redis session max_concurrency limit from 6 to 32

# Description (*)

The default redis session concurrency limit is fairly low. This sometimes triggers error pages for customers who might have opened multiple links in new tabs. 

I see no real drawback increasing this default value from 6 to 32. 
For us this solved the problem completely in production systems. 

Additional investigation is described in https://github.com/magento/magento2/issues/17310#issuecomment-410333850 and https://github.com/magento/magento2/issues/17310#issuecomment-410672582

### Fixed Issues (if relevant)
1. Fixes magento/magento2#17310 that was wrongly closed by a now deleted user.

### Manual testing scenarios (*) copied from @chris-pook 
1. Using Google Chrome, login to the admin area
2. Open 10 or more browser tabs with the orders grid page
3. Click the left most tab
4. Hold shift and click the right most tab (this selects all tabs)
5. Right click the right most tab and select "reload"
6. Check the tabs after they have all reloaded, only ~6 will successfully load, the rest will exhibit this error *
7. Check the var/report folder for the start_session error report relating to the tabs that failed.
8. Now modify the max_concurrency to a value greater than the number of tabs, reload them all again and none will fail.


### Contribution checklist (*)
